src/fu-util.c

```diff
@@ -2289,6 +2364,9 @@ main (int argc, char *argv[])
 		{ "show-all-devices", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all_devices,
 			/* TRANSLATORS: command line option */
 			_("Show devices that are not updatable"), NULL },
+		{ "filter", '\0', 0, G_OPTION_ARG_STRING, &filter,
+			/* TRANSLATORS: command line option */
+			_("Only use devices that include or exclude this flag"), NULL },
```

Comment:
Worth mentioning that it's comma delimited here?

Reply:
I struggled to keep it short -- what about `Filter devices with a comma-delimited set of flags`?

Reply:
I think that covers more, but then you still have the inversion not documented, and where to find the various supported flags not documented.  This feels like it should actually be a few lines.

Reply:
`Filter with a set of device flags using a ~ prefix to exclude, e.g. 'internal,~needs-reboot'`

Reply:
That's concise and good!